migrations/educandu-2021-11-17-02-migrate-cdn-resoures.js

```diff
@@ -0,0 +1,106 @@
+// eslint-disable-next-line camelcase
+
```

Comment:
Could you rename this file to `migrations/educandu-2021-11-18-02-migrate-cdn-resoures.js` so that it comes after `migrations/educandu-2021-11-18-01-add-document-cdn-resources.js` which is added here https://github.com/educandu/educandu/pull/34/files ?
Otherwise they are run in the wrong order.

Reply:
Ok

Reply:
Done :)